group_vars/all.yml

```diff
@@ -1,16 +1,9 @@
 ansible_user: root
-key_path: 'ssh_keys/'
-ssh_users_admins:
```

Comment:
Ich bin SEHR dagegen die SSH-Keys in der adfc_secrets branch zu verschieben. Auch verstehe ich noch nicht wie das funktionieren soll. Wir haben gerade den Vorteil, dass wir mit git wissen, wer was geändert hat. Das ist bei Zugriffsrechten sehr wichtig. Ich möchte wissen wer welchen key wann löscht. Und da hier nur Public-keys sind, ist das auch nicht geheim